example/docker-compose/grafana/dashboards/agent-operational.json

```diff
@@ -916,7 +916,7 @@
                "thresholds": [ ],
                "timeFrom": null,
                "timeShift": null,
-               "title": "Series/Config",
+               "title": "Series per Config",
```

Comment:
It does look a bit worse when we have `Heap Used /Series/Pod` - looks like some kind of resource path or hierarchy. Since `per` fits on the screen, I'd prefer to use that.

Reply:
That's true, let's try out the "per" notation - I'd be happy with it